File: stress/t/flt_dot.c
#include <stdio.h>
#include <math.h>
int main(void){
  long i,n=32; double dot,p; double u[64],v[64];
  for(i=0;i<n;i++){ u[i]=(double)i*0.5-1.0; v[i]=(double)i*0.1+2.0; }
  dot=0.0;
  for(i=0;i<n;i++){ p=u[i]*v[i]; dot=dot+p; }
  printf("dot= %f\n",dot);
  return 0;
}

File: stress/t/flt_dot.f
      PROGRAM FDOT
      IMPLICIT DOUBLE PRECISION (A-H,O-Z)
      INTEGER*8 I,N
      DIMENSION U(0:63),V(0:63)
      N=32
      DO 10 I=0,N-1
        U(I)=DBLE(I)*0.5D0-1.0D0
        V(I)=DBLE(I)*0.1D0+2.0D0
   10 CONTINUE
      DOT=0.0D0
      DO 20 I=0,N-1
        P=U(I)*V(I)
        DOT=DOT+P
   20 CONTINUE
      WRITE(*,'(A,F0.6)') 'dot= ',DOT
      END
